xgi/generators/uniform.py

```diff
@@ -277,7 +278,16 @@ def uniform_erdos_renyi_hypergraph(n, m, p, p_type="degree", seed=None):
         Mean expected degree if p_type="degree" and
```

Comment:
Maybe I'd but the case "prob" first as it's the most classic one, and maybe add "(Default)" next to it too.

Reply:
Fixed.